### DISCUSSION TITLE
The drazzy.com website is down.

### DISCUSSION THREAD

**Discussion by j4m13c0** [SpenceKonde/ATTinyCore#804] [General]
Hi, 
I have been trying to install the boards file into a new install of Arduino and the URL is offline. 
Is it something that is going to be fixed? 

Thanks
Jamie

**Comment by SpenceKonde** [comment]
Yes I have been working on a webserver misconfiguration that I was mistaken about the state of several times and my initial fix is wjat took the webserver down.
